WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -369,12 +369,14 @@ - (BOOL)fb_dismissKeyboardWithKeyNames:(nullable NSArray<NSString *> *)keyNames
     if (nil != auditTypeValue) {
       auditType = valuesToNamesMap[auditTypeValue] ?: [auditTypeValue stringValue];
     }
+    id<FBXCElementSnapshot> snapshot = [extractIssueProperty(issue, @"element") lastSnapshot];
+    NSDictionary *element = snapshot ? [self.class dictionaryForElement:snapshot recursive:NO] : @{};
+                                        
     [resultArray addObject:@{
       @"detailedDescription": extractIssueProperty(issue, @"detailedDescription") ?: @"",
       @"compactDescription": extractIssueProperty(issue, @"compactDescription") ?: @"",
       @"auditType": auditType,
-      @"element": [extractIssueProperty(issue, @"element") description] ?: @"",
-      @"elementDescription": [extractIssueProperty(issue, @"element") debugDescription] ?: @"",
+      @"element": element ?: @{},
```

Comment:
Please do not remove existing properties as it would be a breaking change. Just add a new property instead